pvl_lambertw.m

```diff
@@ -55,7 +55,7 @@
    dw = c2./(c1.*w1 - ((w + 2).*c2./(2*w1)));
    w = w - dw;
 
-   if all(abs(dw) < 0.7e-16*(2+abs(w)))
+   if all(abs(dw) < eps*(2+abs(w)))
```

Comment:
For me `eps = 2.2204e-16` which is greater than the constant 0.7e-16 that is used.  Maybe `eps` is version/platform specific (I'd be surprised)?  I'm not seeing a difference in convergence for z = 4.053154

Reply:
Oh nvm, that value illustrates your point.  We're cycling in and out of machine precision.  Good catch and thanks for the PR!